common/src/main/java/org/opensearch/ml/common/input/MLInput.java

```diff
@@ -55,6 +58,8 @@ public class MLInput implements Input {
     public static final String TARGET_RESPONSE_POSITIONS_FIELD = "target_response_positions";
     // Input text sentences for text embedding model
     public static final String TEXT_DOCS_FIELD = "text_docs";
+    // Input text pairs for text similarity model
+    public static final String TEXT_PAIRS_FIELD = "text_pairs";
```

Comment:
Can we reuse text_docs to keep the interface simpler?

Reply:
@ylwu-amzn see updated interface